client/src/index.js

```diff
@@ -1,21 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux'
-
-import './index.css';
+import './stylesheets/index.scss';
+// import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import { createStore } from 'redux';
-import rootReducer from './store/reducers'
-
+import rootReducer from './store/reducers';
 const store = createStore(rootReducer)
 
 ReactDOM.render(
-  <React.StrictMode>
-      <Provider store={store}>
-        <App />
-      </Provider>,
```

Comment:
If you dont want to set up redux for now (which is perfectly fine), you can also delete all the related imports